views/items/index.pug

```diff
@@ -20,12 +20,15 @@ block content
   div.main-page
     .work-page
       .work-page__main
-        .icon.icon__star
+        .icon.icon__star(data-id=root.id, data-starred=`${root.starred}`, class={starred: root.starred})
```

Comment:
@EthanJStark  Check out the *class attributes* section on the [pug js docs](https://pugjs.org/language/attributes.html) for details - an object can be provided for the class attribute, where the values can be truthy or falsy.  If falsy, that class will not be added to the list of classes for the element.